app/browser/reducers/spellCheckerReducer.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const Immutable = require('immutable')
+const appConstants = require('../../../js/constants/appConstants')
+const {makeImmutable} = require('../../common/state/immutableUtil')
+const {getWebContents} = require('../webContentsCache')
+const spellChecker = require('../../spellChecker')
+
+const migrate = (state) => {
```

Comment:
the other data migrations are in session store. We don't want it to get too bloated, but if we don't remove `dictionary` from the saved state this migration will run on every restart